cypress/integration/page.spec.js

```diff
@@ -2,11 +2,24 @@
 /// <reference types="../support" />
 
 const page = {
-  mockUsers: () => cy.intercept('**/users', { fixture: 'users' }).as('usersRequest'),
-  mockUser1Posts: () => cy.intercept('**/posts?userId=1', { fixture: 'user1Posts' }).as('user1PostsRequest'),
-  mockUser2Posts: () => cy.intercept('**/posts?userId=2', { fixture: 'user2Posts' }).as('user2PostsRequest'),
-  mockPost1Comments: () => cy.intercept('**/comments?postId=1', { fixture: 'post1Comments' }).as('post1CommentsRequest'),
-  mockPost2Comments: () => cy.intercept('**/comments?postId=2', { fixture: 'post2Comments' }).as('post2CommentsRequest'),
+  mockUsers: () =>
+    cy.intercept('**/users', { fixture: 'users' }).as('usersRequest'),
+  mockUser1Posts: () =>
+    cy
+      .intercept('**/posts?userId=1', { fixture: 'user1Posts' })
+      .as('user1PostsRequest'),
+  mockUser2Posts: () =>
+    cy
+      .intercept('**/posts?userId=2', { fixture: 'user2Posts' })
+      .as('user2PostsRequest'),
+  mockPost1Comments: () =>
+    cy
+      .intercept('**/comments?postId=1', { fixture: 'post1Comments' })
```

Comment:
There is a missing semicolon at the end of the `cy.intercept` statement. Ensure that each statement is properly terminated to avoid syntax errors.